templates/download/desktop/thank-you.html

```diff
@@ -40,240 +41,328 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Desktop</h1>
-      <hr>
-      <p>If you're already running Ubuntu, you can upgrade in a few clicks from the Software Updater.</p>
+    <div class="col">
+      {% include 'shared/_server-download-newsletter.html' %}
     </div>
   </div>
 </section>
 
-<section class="p-strip">
-  <div class="row p-divider">
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Verify your download</h3>
-      <p>If you want to verify the  data integrity and authenticity of your Ubuntu download, follow our guide.</p>
-      <p><a href="/tutorials/tutorial-how-to-verify-ubuntu">Learn how to verify&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Ubuntu Desktop guide</h3>
-      <p>Read the official Ubuntu Desktop documentation.</p>
-      <p><a href="https://help.ubuntu.com/stable/ubuntu-help">Ubuntu documentation&nbsp;&rsaquo;</a></p>
+<section class="p-section">
+  <hr class="p-rule is-fixed-width" />
+  <div class="row">
+    <div class="col-9">
+      <div class="p-section--shallow">
+        <h2 class="p-muted-heading">Resources</h2>
+      </div>
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5"><a href="/tutorials/install-ubuntu-desktop">Install Ubuntu Desktop</a></h3>
+          <div>
+            <p>Follow this tutorial to install Ubuntu Desktop on your laptop or PC.</p>
+            <p>You can also run Ubuntu from a USB to try it without installing.</p>
+          </div>
+        </div>
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5"><a href="/tutorials/how-to-run-ubuntu-desktop-on-a-virtual-machine-using-virtualbox">How to run Ubuntu Desktop using VirtualBox</a></h3>
+          <p>Run Ubuntu Desktop in a virtual machine using VirtualBox. A quick start guide that will work across any operating system.</p>
+        </div>
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5"><a href="/tutorials/how-to-install-ubuntu-desktop-on-raspberry-pi-4">How to install Ubuntu Desktop on Raspberry Pi 4</a></h3>
+          <p>A complete guide to installing Ubuntu Desktop on a Raspberry Pi 4 (2GB or above).</p>
+        </div>
+      </div>
     </div>
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Help is always at hand</h3>
-      <p>Join our global community for support and tips on how to get started with Ubuntu.</p>
-      <ul class="p-list">
-          <li><a href="https://discourse.ubuntu.com/">Ubuntu Discourse&nbsp;&rsaquo;</a></li>
-          <li><a href="https://askubuntu.com/">Ask Ubuntu&nbsp;&rsaquo;</a></li>
-          <li><a href="https://answers.launchpad.net/ubuntu">Launchpad Answers&nbsp;&rsaquo;</a></li>
-          <li><a href="https://wiki.ubuntu.com/IRC/ChannelList">IRC-based support&nbsp;&rsaquo;</a></li>
+    <div class="col-3">
+      <div class="p-section--shallow">
+        <h2 class="p-muted-heading">Help is always at hand</h2>
```

Comment:
On the design screen shot it looks like the hr above this should be col-3 instead of full width that appears now

Reply:
wow what kind of screen do you have 😮 ? Or did you just increase contrast?